src/main/java/org/apache/mesos/scheduler/plan/Status.java

```diff
@@ -4,9 +4,9 @@
  * Status of the block. Useful for reporting
  */
 public enum Status {
-  ERROR,
-  WAITING,
-  PENDING,
-  IN_PROGRESS,
-  COMPLETE
+  ERROR, // execution experienced an error
```

Comment:
Can you convert these to JavaDoc style comments `/** */` while you are at it ? That way these will become part of our JavaDocs.